lib/kb_Metrics/metricsdb_controller.py

```diff
@@ -226,7 +258,7 @@ def update_narratives(self, requesting_user, params, token):
         print('Retrieved {} narratives record(s)'.format(len(narr_list)))
         idKeys = ['object_id', 'workspace_id']
         otherKeys = ['name', 'last_saved_at', 'last_saved_by', 'numObj',
-                     'deleted', 'object_version', 'nice_name', 'latest', 'desc']
+                     'deleted', 'object_version', 'nice_name', 'desc']
         for n_data in narr_list:
             filterByKey = lambda keys: {x: n_data[x] for x in keys}
```

Comment:
Are you planning to fix this inline lambda func?

Reply:
Not at this moment, although pep8 does not like it.  It works.

Reply:
So you should fix it.

Reply:
The purpose for this PR is to clean up your previous codes that are not very pythonic. Ideally you should fix all pep8 warnnings.

Reply:
I will later.